docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Unispectral Monarch Pro
+sidebar_label: Unispectral Monarch Pro
+keywords:
+  - unispectral
+  - monarch
+  - multispectral
+  - camera
+  - integration
+  - leo
+  - example
+description: >-
+  Learn how to connect a Monarch Pro multispectral camera to your Leo Rover for
+  multispectral imaging.
+
+image: /img/robots/leo/integrations/monarch-pro/monarch-pro.webp
+---
+
+This tutorial will guide you through the process of connecting a Monarch Pro
+camera to your Leo Rover.
+
+The [Monarch Pro](https://unispectral.com/monarch-pro/) is the ultimate camera
+for advanced imaging, offering detailed material analysis through hyperspectral
+imaging capabilities. It combines high-resolution spectral data capture with
+real-time processing to provide comprehensive insights beyond the visible
+spectrum. It can be used for material identification, crop health monitoring,
+and industrial inspection. With its ability to capture and analyze a wide range
+of wavelengths, the Monarch Pro enables precise analysis and decision-making
+across multiple industries.
+
+<ImageZoom
+  src="/img/robots/leo/integrations/monarch-pro/monarch-pro.webp"
+  width="1200"
+  height="668"
+  figStyle={{
+    width: 600,
+  }}
+/>
+
+## What to expect?
+
+After completing this tutorial, you will have successfully connected a Monarch
+Pro stereo camera to the Leo Rover in ROS, and you will possess basic knowledge
+about the use of multispectral camera.
+
+## Prerequisites
+
+<LinkButton docId="leo-rover/guides/ssh" />
+<LinkButton docId="leo-rover/guides/connect-to-network" />
+<LinkButton docId="leo-rover/advanced-guides/install-ros-on-your-computer" />
+
+## List of components
+
+- Monarch Pro Multispectral Camera
+- Any mounting solution
+- USB-A to USB-C cable
+
+## Software integration
+
+We have prepared special ROS wrapper for the unispectral's Monarch Pro camera.
+
+### Installing the packages
+
+First you have to download it into `src` directory of your ROS workspace:
+
+```bash
+cd <path_to_your_ros_workspace>/src
+git clone https://github.com/fictionlab/ros_monarch_pro.git
```

Comment:
What if the user does not have a directory for a local ROS workspace?

Reply:
Fair point - added a note about the need of having one and a link to our tutorial on how to create one.